camel-openmrs-fhir/src/main/java/org/openmrs/eip/fhir/spring/OpenmrsFhirConfiguration.java

```diff
@@ -42,28 +44,29 @@ public boolean isOauthEnabled() {
 		return isOauthEnabled;
 	}
 	
+	@Bean
+	@Qualifier("openmrsFhirClient")
+	IGenericClient openmrsFhirClient() {
```

Comment:
Is there a reason why these are not public?

Reply:
`openmrsFhirClient` does not need to be public because it is only used within the `OpenmrsFhirConfiguration` class, not intended to publicly exposed and I don't want it to be misused. I could ask the reverse, why public?